website/content/docs/ecs/get-started/migrate-existing-tasks.mdx

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+page_title: Migrate Existing Tasks - AWS ECS
+description: >-
+  Migrate Existing Tasks
+---
+
+# Migrate Existing Tasks
+
+This guide explains how to migrate your existing ECS Tasks to use our [`mesh-task` Terraform module](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-task).
```

Comment:
```suggestion
This topic describes how to migrate your existing ECS Tasks to use our [`mesh-task` Terraform module](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-task).
```
Super minor thing up and totally up for discussion, but I think it would be a good idea to refer to this type of content as 'topics' so we can make it a little clearer internally that we mean a general procedure or process in the documentation. This would be opposed to a larger set of procedures targeting a specific goal, which is how I think of 'tutorials'.

Reply:
I had this exact thought! "Not sure if this is a guide or a tutorial or what" 😄